### PR TITLE
Enable CentOS Advanced Virtualization repositories

### PR DESCRIPTION
**Pull request by nmagnezi**
This is done to get Libvirt7.x and overcome swtpm issues
that currently fail CI.

### CODE REVIEW

**Review by eliorerz — APPROVED** [review]
LGTM

File: scripts/install_environment.sh
Comment (osherdp) [inline scripts/install_environment.sh:24]: @nmagnezi what do you say about adding a comment that this is for a newer libvirt version (<=7)?

**Review by osherdp — CHANGES_REQUESTED** [review]

File: scripts/install_environment.sh
Comment (nmagnezi) [reply scripts/install_environment.sh:24]: No problem 👍 
Let's just wait for CI logs to see that it works as expected.

**Comment by nmagnezi** [comment]
@osherdp [this log](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_assisted-test-infra/1406/pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-tpmv2/1487807103144824832) indicates that the newer libvirt does not solve the issue we have. It does install Libvirt-7.x but we get the same error.

**Comment by osherdp** [comment]
So I guess we can close it
/close

Also we have a new lead with the maintainer of the package: https://github.com/stefanberger/swtpm/issues/644

**Pull request closed without merging**
